data_rows/QueryHistory.js

```diff
@@ -0,0 +1,147 @@
+import uuid4 from "uuid/v4";
+
+import DataRow from "./DataRow";
+import { LoginHistory } from "./LoginHistory";
+import { TYPES } from "./Types";
+import * as helpers from "../helpers";
+
+/**
+ * QUERY_HISTORY View
+ *
+ * https://docs.snowflake.net/manuals/sql-reference/account-usage/query_history.html
+ */
+export class QueryHistory extends DataRow {
+  constructor(databaseName, date) {
+    super();
+    this.QUERY_ID = uuid4();
+    this.setQueryText();
+    this.DATABASE_NAME = databaseName;
+    const queryType = helpers.randomFromArray(this._getQueryTypes());
+    this.QUERY_TYPE = queryType.type;
+    const user = helpers.randomFromArray(LoginHistory.getUsers());
+    this.USER_NAME = user.name;
+    [this.WAREHOUSE_NAME, this.WAREHOUSE_SIZE] = helpers.randomFromArray([
+      ["BIG_WH", "X-Large"],
+      ["SMALL_WH", "Small"]
+    ]);
```

Comment:
fancy

Reply:
are you referring to my super original names/sizes or the "de-structuring" feature of JS (python's unpacking, php's `list()`)? I'm happy to see JS has it too :-)